ec2/spark_ec2.py

```diff
@@ -47,6 +48,8 @@
 else:
     from urllib.request import urlopen, Request
     from urllib.error import HTTPError
+    raw_input = input
+    xrange = range
```

Comment:
Hmm, this part smells. Why are we doing this here? And does it do the right thing on Python 2.6 (the earliest version spark-ec2 supports)?